### DISCUSSION TITLE
Where to find new model?

### DISCUSSION THREAD

**Discussion by owen555999** [upscayl/upscayl#497] [Q&A] [answered]
i want to use any new model
so where can i find all new model?

**Comment by aaronliu0130** [comment, marked as answer]
https://github.com/upscayl/upscayl/wiki/🖥%EF%B8%8F-Model-Conversion-‐-Create-more-AI-models!

> You can get models from https://openmodeldb.info/.

**Comment by NayamAmarshe** [comment]
We have a custom models repository where you can find several models that you can use.

Here's the link to the guide: https://github.com/upscayl/upscayl/wiki/Guide#custom-models
